desktop/com.github.xournalpp.xournalpp.appdata.xml

```diff
@@ -65,8 +65,13 @@
   </screenshots>
   <launchable type="desktop-id">com.github.xournalpp.xournalpp.desktop</launchable>
   <provides>
-    <id>xournalpp.desktop</id>
+    <binary>xournalpp</binary>
```

Comment:
first of all: the `provides` section is optional.
It exists to ease transition to the new app id.
The old appid was `xournalpp.desktop`.
Changing it to a name of a binary is semantically not as correct as the previous appid.

Reply:
From the [Section 2.1.1 of the specification](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#sect-Metadata-GenericComponent):
> If a metainfo file contains a &lt;provides/&gt; tag, distributors must also ensure that the package providing the file contains all items referenced by that statement, or is installed by a metapackage depending on packages which provide these items.

Since `xournalpp.desktop` is an old id, it will presumably not be installed anymore. In either case, we should still include the binary tag because `xournalpp` _is_ installed.

Reply:
Nevermind, I missed this bit ([Section 2.1.3](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-provides)):
> Contains the component-ID of another software component. The presence of this tag indicates that the software component containing it is able to provide all functionality of the one referenced in the &lt;provides/&gt; ↪ &lt;id/&gt; tag.
> This is useful in case a component-id had to be renamed in the past, e.g. because its domain name changed.

So the revision is to not remove the provides for `xournalpp.desktop`.